src/components/apps/editor/ParamSelectionPalette.js

```diff
@@ -0,0 +1,274 @@
+                    2.3
+                </Typography>
+            );
+            break;
+
+        case AppParamTypes.TEXT_SELECTION:
+        case AppParamTypes.INTEGER_SELECTION:
+        case AppParamTypes.DOUBLE_SELECTION:
+        case AppParamTypes.REFERENCE_GENOME:
+        case AppParamTypes.REFERENCE_SEQUENCE:
+        case AppParamTypes.REFERENCE_ANNOTATION:
+            icon = <ListIcon color="action" />;
+            break;
+
+        default:
+            icon = <Code color="action" />;
+            break;
+    }
+
+    return (
+        <ListItemAvatar>
+            <Avatar>{icon}</Avatar>
+        </ListItemAvatar>
+    );
+}
+
+function ParamTypeListItem(props) {
+    const { id, paramType, onClick } = props;
+
+    const [helpOpen, setHelpOpen] = React.useState(false);
+
+    const { t } = useTranslation([
+        "app_param_types",
+        "app_editor",
+        "app_editor_help",
+    ]);
+
+    return (
+        <>
+            <ListItem id={id} divider={!helpOpen} button onClick={onClick}>
+                <ParamTypeIcon paramType={paramType} />
+                <ListItemText primary={t(paramType)} />
+                <ListItemSecondaryAction>
+                    <IconButton
+                        id={buildID(id, t("app_editor:helpText"))}
+                        edge="end"
+                        aria-label={t("app_editor:helpText")}
+                        onClick={() => setHelpOpen(!helpOpen)}
+                    >
+                        <HelpIcon color="primary" />
+                    </IconButton>
+                </ListItemSecondaryAction>
+            </ListItem>
+            <Collapse in={helpOpen} timeout="auto" unmountOnExit>
+                <ListItem divider>
+                    <ListItemText
+                        secondary={
+                            <Trans
+                                t={t}
+                                i18nKey={`app_editor_help:${paramType}`}
+                                components={{
+                                    b: <b />,
+                                    i: <i />,
+                                    p: <p />,
+                                    support: (
+                                        <Link
+                                            id={buildID(
+                                                id,
+                                                UtilIds.CONTACT_SUPPORT_BUTTON
+                                            )}
+                                            component="button"
+                                            variant="body2"
+                                            onClick={(event) => {
+                                                // prevent parameter selection
+                                                event.stopPropagation();
+                                                intercomShow();
+                                            }}
+                                        />
```

Comment:
By the way, setting the `variant` prop on a `Link` that is also using `component="button"` will fix the font appearance (in Firefox at least), when set to the same `Typography` variant as the surrounding text.

[The docs](https://material-ui.com/components/links/#simple-links) say that the "Link component is built on top of the Typography component. You can leverage its properties."